ko/sql/schema/index_stmt.rst

```diff
@@ -224,6 +237,71 @@ CREATE INDEX
 |                                                                   |                                                                                   |
 +-------------------------------------------------------------------+-----------------------------------------------------------------------------------+
 
+.. _deduplicate_overview:
+
+DEDUPLICATE 
+-----------
+
+**DEDUPLICATE** 옵션을 사용하면 특정 키값에 치우쳐진 비대칭 데이터(skewed data)에 의해 인덱스 데이터 수정 시 발생할 수 있는 성능 저하를 개선할 수 있다. 이 옵션 값으로 특정 키 값에 대한 OID 오버플로우 페이지의 연결 리스트가 너무 길게 만들어지지 않게 완화하도록 조정할 수 있으며 그에 따라 삽입/삭제/갱신 및 vacuum 성능을 개선한다. 하지만 시스템에 의해 숨겨진 인덱스 칼럼이 추가되었으므로 인덱스의 단말 노드의 개수와 트리의 높이가 증가 할 수 있어 인덱스의 크기가 커지며 조회 성능에 영향을 줄 수 있으므로 주의헤야 한다. 특히, 인덱스 데이터가 키 값에 대하여 균일하게 분포하는 경우에는 성능 개선 효과 없이 인덱스의 크기만 커질 수 있으므로 주의해야 한다.
+
+*deduplicate level*\의 값이 1 이상 지정되면 인덱스 생성 시 사용자가 명시한 인덱스 칼럼에 더하여 시스템 내부적으로 사용하는 숨겨진 인덱스 칼럼이 하나 추가되는데, 이 값을 사용하여 특정 키값에 치우치지 않도록 중복도를 완화한다. *deduplicate level*\ 값이 커질수록 중복도를 더 완화하게 되며, 특정 키값의 오버플로우 페이지의 연결 리스트 길이가 짧아지게 된다.
+
+.. note::
+        
+    * 일반적으로 시스템이 부여할 수 있는 키 값의 개수는 *deduplicate level*\이 1씩 증가할 때마다 2배씩 늘어난다. 따라서 오버플로우 페이지의 연결 리스트 길이는 반씩 줄어들 것이다.
+    * 시스템이 부여하는 키 값은 해당 레코드의 OID 정보 중에서 페이지 번호에 대한 나머지 연산을 통해 얻는다. 따라서 실질적으로 시스템이 부여 해주는 키 값의 분포는 레코드가 저장된 페이지 번호에 의해 결정된다.
+    * 오버플로우 페이지의 연결 리스트는 조회 성능에서는 더 유리하므로 *deduplicate level*\을 무조건 크게 해서 오버플로우 페이지 자체가 생성되지 않도록 유도하는 것은 바람직하지 않으며 중복도를 고려해서 적당한 값을 지정해야 한다. (하나의 키에 대한 OID 오버플로우 페이지의 길이는 수십 혹은 수백 페이지 이내가 되도록 조정한다)
```

Comment:
```suggestion
    * 오버플로우 페이지 연결 리스트는 추가, 삭제, 갱신 성능 면에서는 불리하지만 조회 성능 면에서는 유리하다. 그러므로,  *deduplicate level*\을 무조건 ...
```

Reply:
Thank you,  I will reflect your opinion.